Classes/Backend/RecordList/NewsDatabaseRecordList.php

```diff
@@ -34,8 +34,15 @@ public function listURL($alternativeId = '', $table = '-1', $excludeList = ''):
         } else {
             $urlParameters['id'] = $this->id;
         }
-        if ($this->thumbs) {
-            $urlParameters['imagemode'] = $this->thumbs;
+        // todo can be remove when 10.4 supported dropped
+        $versionInformation = GeneralUtility::makeInstance(\TYPO3\CMS\Core\Information\Typo3Version::class);
```

Comment:
No need to use GU, just `(new \TYPO3\CMS\Core\Information\Typo3Version())->getMajorVersion()` as this isn't a service.